README.md

```diff
@@ -7,14 +7,13 @@ Hazel is primarily an early-stage interactive application and rendering engine f
 ## Getting Started
 Visual Studio 2017 or 2019 is recommended, Hazel is officially untested on other development environments whilst we focus on a Windows build.
 
-You can clone the repository to a local destination using git:
+Start by cloning the repository with `git clone --recursive https://github.com/TheCherno/Hazel`.
 
-`git clone --recursive https://github.com/TheCherno/Hazel`
-
-Make sure that you do a `--recursive` clone to fetch all of the submodules!
+If the repository was cloned non-recursively previously, use `git submodule update --init` to clone the necessary submodules.
+Hazel uses _Premake 5_ as a build generation tool. Visit the [Premake website](https://premake.github.io/download.html) to download and install it.
```

Comment:
There is no need to download, nor to install premake. It comes within the repo already and can be used stand alone

Reply:
I think i have now fixed that